spec/smoke_test/job_seekers_can_view_homepage_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'browser_test_helper'
+require 'pry'
+
+RSpec.describe 'Page availability', js: true do
+  context 'Job seeker visits home page' do
+    it 'Page should be available' do
+      page = Capybara::Session.new(:poltergeist)
+      page.visit 'https://teaching-vacancies.service.gov.uk/'
+      expect(page).to have_content('Find a job in teaching')
+    end
+
+    it 'Job search should return a result' do
```

Comment:
```suggestion
    it 'should return a result when performing a job search' do
```
perhaps this might sound better?

Reply:
yep, thanks! Resolved in a255f3e